src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1787,6 +1787,36 @@ public function testAllExcludesDisabledLazyCommand()
         $this->assertArrayNotHasKey('disabled', $application->all());
     }
 
+    public function testGetShellVerbosity()
+    {
+        $application = new Application();
+        $this->assertSame(-1, $application->getShellVerbosity(new ArrayInput(['--quiet'])));
```

Comment:
I think that it would be easier to read by using a `@dataProvider` for the tests before `putenv('SHELL_VERBOSITY=3');`.

Reply:
Good Idea. I'll make the change if I can find the time

Reply:
@alexislefebvre Added a data provider for the tests. Thanks for the suggestion